services/catalog/rs_server_catalog/main.py

```diff
@@ -210,6 +211,8 @@ async def dispatch(self, request: Request, call_next: Callable):
 app = api.app
 app.openapi = extract_openapi_specification
 
+# Configure OpenTelemetry
+opentelemetry.init_traces(app, "rs.server.catalog")
```

Comment:
That's all you need to do to see the span traces in tempo. No need to manually add traces to your functions or endpoints.